src/main.ml

```diff
@@ -261,6 +262,11 @@ let warning_OptionalArgumentNotErased err errLines =
   let line = int_of_string (get_match lineR err) in
   let chars1 = int_of_string (get_match chars1R err) in
   let chars2 = int_of_string (get_match chars2R err) in
+  (* Hardcoding 16 for now. We might one day switch to use the variant from
+  https://github.com/ocaml/ocaml/blob/901c67559469acc58935e1cc0ced253469a8c77a/utils/warnings.ml#L20 *)
+  let optionalErasureWarningR = {|Warning 16: this optional argument cannot be erased\.|} in
+  (* Using get_match_n because there's nothing to capture from the error message. *)
+  let _ = get_match_n 0 optionalErasureWarningR err in
```

Comment:
This will cause an unused variable warning, and won't warn a second time (because... make? ocaml?). So usually you'd use `ignore`